build.gradle

```diff
@@ -823,6 +823,7 @@ project(':core') {
     implementation libs.scalaLogging
     implementation libs.slf4jApi
     implementation(libs.zookeeper) {
+      implementation libs.dropwizardMetrics
```

Comment:
Can you add a comment explaining that this is needed for Apache ZooKeeper, and should NOT be used by Kafka itself?

Reply:
Good point -- added.